yarn.lock

```diff
@@ -3819,6 +3819,7 @@
 "@patternfly/patternfly@2.65.3":
```

Comment:
Why is the yarn.lock being updated?

Reply:
Yarn install. On Slack Zack said it was ok to commit these.

Reply:
I don't see any harm, but there's really no need for this file to change, unless we're introducing new dependencies.